atqa-framework/utils/copyDirHandler.ts

```diff
@@ -0,0 +1,14 @@
+import fs from 'fs-extra';
+import { Folder } from '../interfaces/Folder.interface';
+
+export const copyDir = async (dir: Folder): Promise<void> => {
```

Comment:
Doesn't make sense to declare this `async`, because it's not.

Reply:
async methods will be added on the next stage. I suggest keep it as is until the coming changes.

Reply:
If you need something on some "next stage", you can use a separate branch, stash or whatever for that